doc/rados/operations/balancer.rst

```diff
@@ -60,11 +60,14 @@ of the following form:
 A larger value may increase the speed of cluster balancing / convergence
 at the potential cost of greater impact on client operations.
 
-There is a separate setting for how uniform the distribution of PGs must be for
-the module to consider the cluster adequately balanced.  At the time of writing
-(June 2025), this value defaults to ``5``, which means that if a given OSD's PG
-replicas vary by five or fewer above or below the cluster's average, it will be
-considered sufficiently balanced.
+The setting ``upmax_max_deviation`` sets the threshold at which the balancer
```

Comment:
Suggest s/setting/option/ so we aren't semi-duplicating "set".
Below suggest s/module// , s/balanced/uniform/